include/PR/gu.h

```diff
@@ -1,26 +1,63 @@
-#ifndef _GU_H_
-#define _GU_H_
-
-typedef struct {
-    /* 0x0 */ unsigned char* base;
-    /* 0x4 */ int fmt;
-    /* 0x8 */ int siz;
-    /* 0xC */ int xsize;
-    /* 0x10 */ int ysize;
-    /* 0x14 */ int lsize;
-    /* 0x18 */ int addr;
-    /* 0x1C */ int w;
-    /* 0x20 */ int h;
-    /* 0x24 */ int s;
-    /* 0x28 */ int t;
-} Image;
-
-
-typedef struct {
-    /* 0x0 */ float col[3];
-    /* 0xC */ float pos[3];
-    /* 0x18 */ float a1;
-    /* 0x1C */ float a2;
-} PositionalLight;
+#ifndef PR_GU_H
+#define PR_GU_H
+
+#include "ultratypes.h"
+#include "gbi.h"
+
+
+#ifndef MAX
+#define MAX(a,b) (((a)>(b))?(a):(b))
```

Comment:
`MAX(` is used in a few places like `z_room`, `en_ge1`, `en_go`, `z_en_ossan`, `z_en_sob1`. Should this macro be limited to libultra or be put in a z64 header?

Reply:
It is fine to use this macro outside of libultra. Since libultra defines this macro then there's no point redefining it in a z64 header